### PR TITLE
Added nbiot possibilty for imbuildings people counter

### PR DESCRIPTION
**Pull request by J2thatsme**

### CODE REVIEW

File: types/IMBuildings/PeopleCounter/uplink.js [hunk 1]
```diff
@@ -12,9 +12,78 @@ function readUInt16BE(payload, index) {
...
+
 function consume(event) {
-  const payload = event.data.payloadHex;
-  const bytes = Hex.hexToBytes(payload);
+  let payload;
+  let bytes;
+
+  // NBIOT
+  if (event.data.reports !== undefined) {
+    payload = event.data.reports[0].value;
```
Comment (mariusgiger) [inline types/IMBuildings/PeopleCounter/uplink.js:78]: loop over the reports and emit each one?

File: types/IMBuildings/PeopleCounter/uplink.spec.js [hunk 2]
```diff
@@ -80,15 +80,68 @@ describe("IMBuilding People counter", () => {
...
         utils.validateSchema(value.data, lifecycleSchema, { throwError: true });
       });
 
       consume(data);
     });
+
+    it("should decode the IMBuilding People NB-IOT payload", () => {
+      const data = {
+        data: {
+          reports: [
```
Comment (mariusgiger) [inline types/IMBuildings/PeopleCounter/uplink.spec.js:97]: this is iotcreators specific right? if yes, how do we support other structures?

**Review by mariusgiger — COMMENTED** [review]
looks good, two comments

File: types/IMBuildings/PeopleCounter/uplink.spec.js (hunk 2 above)
Comment (J2thatsme) [reply types/IMBuildings/PeopleCounter/uplink.spec.js:97]: As this is the default way. I suppose we keep it that way. I cant really work with structures i don't know

File: types/IMBuildings/PeopleCounter/uplink.js (hunk 1 above)
Comment (mariusgiger) [reply types/IMBuildings/PeopleCounter/uplink.js:78]: please add this as a comment then

File: types/IMBuildings/PeopleCounter/uplink.spec.js (hunk 2 above)
Comment (mariusgiger) [reply types/IMBuildings/PeopleCounter/uplink.spec.js:97]: We are going to provide an out of the box support for iotcreators which will simplify the setup

**Review by mariusgiger — DISMISSED** [review]

**Pull request merged**